### DISCUSSION TITLE
Mercator Ikuu - 2/5m light strips including RGB and tri-colour bulbs

### DISCUSSION THREAD

**Discussion by jchurchward** [Koenkk/zigbee2mqtt#9043] [Devices]
Hi All,

Anyone working on these Mercator Ikuu devices that are sold in Australia? I see support for Ikuu is growing but these are still yet to be added.

Fingerprint:  _TZ3000_cmaky9gq
Link: https://www.lightingillusions.com.au/shop/d-i-y/5-metre-mercator-smart-zigbee-12v-rgb-cct-led-strip-light-kit-smld533-zb-42776

Fingerprint: _TZ3000_x2fqbdun
Link: https://www.lightingillusions.com.au/shop/globes/rgb-and-tri-colour-mercator-smart-zigbee-9-5w-classic-a60-led-globe-edison-screw-e27-s9e27led9w-rgb-z-42746

Fingerprint: _TZ3000_tza2vjxx
Link: https://www.lightingillusions.com.au/shop/d-i-y/2-metre-mercator-smart-zigbee-12v-rgb-cct-led-strip-light-kit-smld233-zb-42774

**Comment by Vertikar** [comment]
I've tried writing an external converter for the globe, but haven't got it working yet. Was going to raise a bug today and see if I anyone can help out. I'll share a link to it once I raise it

**Reply by jchurchward** [reply]
Excellent news. Yes please share what your working on too.

**Reply by Vertikar** [reply]
Here's the issue I've opened: https://github.com/Koenkk/zigbee2mqtt/issues/9099

**Reply by Vertikar** [reply]
It's been added to the Dev branch, i'm still using the external converter for now

**Reply by jchurchward** [reply]
It does work with the LED strips it's only a couple of colours that are out. Awesome!

**Reply by jchurchward** [reply]
I have requested support to be added for the RGB+CCT LED light strips [here](https://github.com/Koenkk/zigbee2mqtt/issues/9316)

**Reply by jchurchward** [reply]
Support has been added. All done :)
